examples/gluon_alexnet/README.md

```diff
@@ -71,13 +91,13 @@ cp gluon_pretrained_alexnet.py synset.txt signature.json /tmp/models
 ```
 3. Run the model-export tool on this folder.
 ```bash
-mxnet-model-export --model-name="pretrained-alexnet" --model-path="/tmp/models" --service-file-path="/tmp/models/gluon_pretrained_alexnet.py"
+model-archiver --model-name alexnet --model-path /tmp/models --handler gluon_pretrained_alexnet:pretrained_gluon_alexnet --runtime python --export-path /tmp
 ```
-This creates a model-archive file `pretrained-alexnet.model`.
+This creates a model-archive file `/tmp/alexnet.mar`.
 
 4. You could run the server with this model file to serve the pre-trained alexnet.
 ```bash
-mxnet-model-server --models alexnet=pretrained-alexnet.model
+mxnet-model-server --start --models /tmp/alexnet.mar --mms-config /tmp/mms_config
```

Comment:
mxnet-model-server --start --models alexnet.mar --model-store /tmp

So you don't need a config file.

Reply:
configuration file is for advanced user. We should keep the example as simple as possible.